src/dbpedia_linker.py

```
@@ -0,0 +1,74 @@
+STOPWORDS = {"fc", "club", "sc", "the", "of", "and", "in", "inc"}
+
+def clean_label(label):
+    label = re.sub('<[^<]+?>', '', label)
+    label = re.sub(r'\s*\(.*?\)', '', label)
+    return label.strip()
+
+def covers_query(query, label):
+    q_tokens = [t for t in query.lower().replace("-", " ").split() if t not in STOPWORDS]
+    l_tokens = label.lower().replace("-", " ").split()
+    for q in q_tokens:
+        if not any(l.startswith(q) for l in l_tokens):
```

Ruff (0.15.0)

[error] Ambiguous variable name: `l`

(E741)